### DISCUSSION TITLE
Stable release ?

### DISCUSSION THREAD

**Discussion by GTrannoy** [stnolting/neorv32#81] [Show and tell]
Can you add a new stable release, please ?

Or, is it planned soon ?

**Comment by stnolting** [comment]
This is related to #37... 😅 
I was thinking to make release `1.5.7.0` somewhere at the beginning of July. But I am open for discussion.

**Reply by umarcor** [reply]
Please, just tag and release `1.5.7`. I.e., do not use the last `.0` in the tags (it's implicit, we know it's always 0).
That's enough for being semver compliant and for closing #37.

**Reply by stnolting** [reply]
:+1: 😄

**Reply by GTrannoy** [reply]
Ok, good news !
